lib/fd_send_recv_stubs.c

```diff
@@ -36,7 +36,7 @@
 
 void raise_error(int code)
 {
-	static value *exn = NULL;
+	static const value *exn = NULL;
```

Comment:
We could use `uerror` to raise a Unix_error, unless we rely on the custom exception here for some reason.

Reply:
Now using `caml_uerror` on both calls that used this function